main.py

```diff
@@ -355,12 +439,31 @@ def __init__(self, *args):
 		self.restartButton.clicked.connect(self.onRestartButtonClicked)
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
+		
+		#Setup Games list
+		self.tableGames.setRowCount(0)
+		self.tableGames.setColumnCount(4)
+		self.tableGames.setColumnWidth(0, 85)
+		self.tableGames.setColumnWidth(1, 177)
+		self.tableGames.setColumnWidth(2, 177)
+		self.tableGames.setColumnWidth(3, 67)
+		self.tableGames.verticalHeader().setVisible(False)
+		self.tableGames.horizontalHeader().setVisible(False)
+		self.tableGames.setShowGrid(False)
+		self.tableGames.setAlternatingRowColors(True)
+		self.tableGames.setStyleSheet("QTableWidget { alternate-background-color: #5E5E5E; background-color: #404040; }"
+			+ "QTableWidget::item:selected { background: #5b9aff; }")
+		self.tableGames.setEditTriggers(QAbstractItemView.NoEditTriggers)
+		self.tableGames.setSelectionBehavior(QAbstractItemView.SelectRows)
+		self.tableGames.setSelectionMode(QAbstractItemView.SingleSelection)
+		self.populateLibrary(self.settings.settings['dvd_folder_path'])
```

Comment:
This should be separated so it's more obvious. You are mixing your UI view, with your data model.